tests/test_pdf_uploader.py

```diff
@@ -0,0 +1,95 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from dsst_etl.upload_pdfs import PDFUploader
+from dsst_etl.db import get_db_session
+from dsst_etl.models import Documents, Provenance, Works
+from pathlib import Path
+
+class TestPDFUploader(unittest.TestCase):
+
+    @patch('dsst_etl.upload_pdfs.boto3.client')
+    def setUp(self, mock_boto_client):
```

Comment:
I needed to manually create the database by running db_init() before these test would pass. Please added init_db() to the unittest.